File: Patterns/08. Inverted Pyramid Patterns/76. Inverted Pyramid Even Numbers/ipen.py
n = int(input("Enter a Number : "))

for i in range(1,n+1):
    # print spaces
    for j in range(1,i):
        print(end='   ')
    # print first half
    for j in range(1,n-i+2):
        print(f"{2*j:3}",end='')
    # print second half
    for j in range(n-i,0,-1):
        print(f"{2*j:3}",end='')
    print()


'''

Output:

  2  4  6  8 10  8  6  4  2
     2  4  6  8  6  4  2
        2  4  6  4  2
           2  4  2
              2

'''

File: Patterns/08. Inverted Pyramid Patterns/76. Inverted Pyramid Even Numbers/ipen.java
import java.util.*;
public class ipen{
    public static void main(String [] args){
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a Number : ");
        int n = sc.nextInt();

        for(int i=1; i<=n; i++){
            // print spaces
            for(int j=1; j<=i-1; j++){
                System.out.print("   ");
            }
            // print first half
            for(int j=1; j<=n-i+1; j++){
                System.out.printf("%3d",2*j);
            }
            // print second half
            for(int j=n-i; j>=1; j--){
                System.out.printf("%3d",2*j);
            }
            System.out.println();
        }
    }
}

/*
Output

  2  4  6  8 10  8  6  4  2
     2  4  6  8  6  4  2
        2  4  6  4  2
           2  4  2
              2

*/